build.gradle

```diff
@@ -14,12 +21,11 @@ sourceCompatibility = 11
 targetCompatibility = 11
 
 ext {
-    grpcVersion                 = '1.32.1'
-    protobufVersion             = '3.13.0'
-    serviceGeneratorVersion     = '3.1.12'
+    grpcVersion = '1.58.0'
```

Comment:
I believe that version `1.56.0` should be used (because the BOM contain that version)